docs/modules/record-encryption/proc-configuring-record-encryption-filter.adoc

```diff
@@ -35,14 +35,20 @@ filters:
       selector: <KEK_selector_service_name> # <3>
       selectorConfig:
         template: "KEK_${topicName}" # <4>
+      experimental: # <5>
+        "encryptionDekExpireAfterWriteSeconds": 7200 # <6>
+        "maxEncryptionsPerDek": 5000000 # <7>
 	 # ...
 ----
 <1> The KMS service name.
 <2> Configuration specific to the KMS provider.
 <3> The Key Encryption Key (KEK) selector to use. The `${topicName}` is a literal understood by the proxy. 
 For example, if using the `TemplateKekSelector` with the template `KEK_$\{topicName}`, create a key for every topic that
 is to be encrypted with the key name matching the topic name, prefixed by the string `KEK_`.
-<4> The template for deriving the KEK, based on a specific topic name. 
+<4> The template for deriving the KEK, based on a specific topic name.
+<5> **Warning**: experimental configurations do not offer compatibility guarantees
```

Comment:
```suggestion
<5> **Warning**: There is no guarantee that options in the experimental section will exist in future releases. Options which are determined to be useful will be promoted to the main config model, other options will be dropped without following a depreciation cycle
```

This is maybe the wrong place to try and document this. I'm also not sure we have made it explicit anywhere that we consider the config model to be public API and thus compliant with semver...

Reply:
mm the other place we document the deprecation policy is in the DEV_GUIDE which isn't so useful for users, it's more aimed at filter authors.

Think it's a good one to ask in chat, whether we need a user-facing policy for our configuration API compatibility and what it should be if so.

Maybe removing the comment from this PR and solving separately is the right thing?

Reply:
Have removed the explanation, better to say nothing while it's undefined